### PR TITLE
add Dynamic Configuration Consul AclToken 

### PR DESCRIPTION
**Pull request by web-xiaxia**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [x] New feature provided
- [ ] Improve performance

- Related issues

___
### Bug fix
- Bug description.

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

File: oap-server/server-bootstrap/src/main/resources/application.yml [hunk 1]
```diff
@@ -223,6 +225,8 @@ configuration:
 #    hostAndPorts: ${consul.address}
 #    # Sync period in seconds. Defaults to 60 seconds.
 #    period: 1
+#    # Consul aclToken
+#    #aclToken: ${consul.aclToken}
```
Comment (wu-sheng) [inline oap-server/server-bootstrap/src/main/resources/application.yml]: If you want this too, you miss some codes change in the configuration model. Your changes are only related to cluster model.

Comment (web-xiaxia) [reply oap-server/server-bootstrap/src/main/resources/application.yml]: 是不用改这个文件么？

Comment (wu-sheng) [reply oap-server/server-bootstrap/src/main/resources/application.yml]: English, please. You are changing two places, one is cluster model, the other is configuration model. Both of them have Consul implementations, but they are totally different.

Comment (web-xiaxia) [reply oap-server/server-bootstrap/src/main/resources/application.yml]: configuration model  and cluster model  have changed

File: oap-server/server-cluster-plugin/cluster-consul-plugin/src/main/java/org/apache/skywalking/oap/server/cluster/plugin/consul/ClusterModuleConsulProvider.java [hunk 2]
```diff
@@ -76,6 +76,10 @@ public ClusterModuleConsulProvider() {
 //                    we should set this value or it will be blocked forever
                     .withConnectTimeoutMillis(3000);
 
+            if (config.getAclToken() != null && !config.getAclToken().isEmpty()) {
```
Comment (dmsolr) [inline oap-server/server-cluster-plugin/cluster-consul-plugin/src/main/java/org/apache/skywalking/oap/server/cluster/plugin/consul/ClusterModuleConsulProvider.java]: Why do not you use `!Strings.isNullOrEmpty(config.getAclToken())`? I think that it is more readable.

Comment (wu-sheng) [reply oap-server/server-cluster-plugin/cluster-consul-plugin/src/main/java/org/apache/skywalking/oap/server/cluster/plugin/consul/ClusterModuleConsulProvider.java]: @web-xiaxia this should be fixed

Comment (web-xiaxia) [reply oap-server/server-cluster-plugin/cluster-consul-plugin/src/main/java/org/apache/skywalking/oap/server/cluster/plugin/consul/ClusterModuleConsulProvider.java]: Replacement completed

**Comment by web-xiaxia** [comment]
> @web-xiaxia could you please also update the documentation? Codes look good to me, thanks

changed

**Pull request merged**